how-to/features/baremetal.rst

```diff
@@ -0,0 +1,378 @@
+Baremetal deployment service
+============================
+
+This feature deploys `Ironic`_, the bare metal provisioning service for
+OpenStack. It allows OpenStack users to provision bare metal machines,
+as opposed to virtual machines.
+
+Enabling Baremetal
+------------------
+
+This feature requires the storage role. To enable this feature, run the
+following command:
+
+::
+
+   sunbeam enable baremetal
+
+.. note::
+   This feature requires the ``microceph`` charm channel to be on
+   ``squid/edge`` or newer.
```

Comment:
Is this still the case? there's been a lot of promotions around microceph channels recently.